src/components/menu/menu.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "./menu.module.scss";
+
+export const Menu = () => {
+  const menu = ["all", "documentary", "comedy", "horror", "crime"];
+  const sortOptions = ["Release date", "Title"];
+
+  const [selectedMenu, setSelectedMenu] = useState("all");
```

Comment:
In TS let's use parametrised hooks, like:
```suggestion
  const [selectedMenu, setSelectedMenu] = useState<string>("all");
```